### PR TITLE
Home Work MODULE 9: BASIC KNOWLEDGE OF TA FRAMEWORKS

### PR DESCRIPTION
**Pull request by KseniiaOvsiannikova**

### CODE REVIEW

File: src/main/java/HelloWebDriver.java [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-            driver.get("http://google.com");
-            driver.quit();
-
-            System.setProperty("webdriver.gecko.driver", "src\\test\\resources\\webdrivers\\geckodriver.exe");
-            driver = new FirefoxDriver();
-            driver.get("http://google.com");
-            driver.quit();
-        }
-    }
+//import org.openqa.selenium.WebDriver;
```
Comment (olelysieev-clgx) [inline src/main/java/HelloWebDriver.java]: please uncomment or remove commented code

File: src/test/java/pages/TutorialspointPage.java [hunk 2]
```diff
@@ -20,14 +22,15 @@ public TutorialspointPage(WebDriver driver) {
...
     public TutorialspointPage openPage() {
+        driver.get(TUTORIALSPOINT_URL);
         waitForPageLoad(driver);
 
         return this;
     }
 
     public TutorialspointPage searchForPhrase() {
         String searchValue = "submit() vs click()";
-        waitForPageLoad(driver);
```
Comment (olelysieev-clgx) [inline src/test/java/pages/TutorialspointPage.java:30]: Why this wait was removed?

File: src/test/java/service/URLCreator.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+package service;
+
+public class URLCreator {
+    private static final String GOOGLE_URL = "testdata.google.url";
+    public static final String WIKIPEDIA_URL = "testdata.wikipedia.url";
```
Comment (olelysieev-clgx) [inline src/test/java/service/URLCreator.java]: private

File: src/test/java/tests/BaseTest.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import org.testng.asserts.SoftAssert;
+import util.TestListener;
+
+import java.util.logging.Logger;
+
+@Listeners({TestListener.class})
+public class BaseTest {
+
+    protected WebDriver driver;
+    protected Logger logger = Logger.getGlobal();
```
Comment (olelysieev-clgx) [inline src/test/java/tests/BaseTest.java]: please use one logger type in your project - log4j

File: src/main/java/HelloWebDriver.java (hunk 1 above)
Comment (KseniiaOvsiannikova) [reply src/main/java/HelloWebDriver.java]: Thank you for pointing, I deleted the class.
Please see changes in the following commit.

File: src/test/java/service/URLCreator.java (hunk 3 above)
Comment (KseniiaOvsiannikova) [reply src/test/java/service/URLCreator.java]: My apologizes, refactored.
Please see in the following commit.

File: src/test/java/tests/BaseTest.java (hunk 4 above)
Comment (KseniiaOvsiannikova) [reply src/test/java/tests/BaseTest.java]: Refactored, thank you.
Please see the changes in the following commit.

**Review by olelysieev-clgx — APPROVED** [review]

**Pull request merged**